lib/ChainwebData/AccountDetail.hs

```diff
@@ -17,9 +19,16 @@ data AccountDetail = AccountDetail
   , _acDetail_name :: Text
   , _acDetail_fromAccount :: Text
   , _acDetail_toAccount :: Text
-  , _acDetail_amount :: Double
+  , _acDetail_amount :: Decimal
```

Comment:
Oh, I think we've been using this to avoid this orphan issue.  https://github.com/kadena-io/chainweb-api/blob/master/lib/Chainweb/Api/ParsedNumbers.hs#L23